algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -21,12 +21,23 @@
     ConformerEncoderDecoder
 from algorithmic_efficiency.workloads.librispeech_conformer.librispeech_pytorch.model import \
     initialize
+from algorithmic_efficiency.workloads.librispeech_conformer.librispeech_pytorch.model import \
+    Subsample
 
-USE_PYTORCH_DDP, RANK, DEVICE, N_GPUS = pytorch_setup()
+USE_PYTORCH_DDP, RANK, DEVICE, N_GPUS = pytorch_utils.pytorch_setup()
 
 MAX_INPUT_LENGTH = 320000
 
 
+def _update_model_dropout(model, residual_dropout_rate, input_dropout_rate):
+  # model.modules() returns the model itself as the first element.
+  for child in list(model.modules())[1:]:
+    if isinstance(child, Subsample):
+      child.input_dropout_rate = input_dropout_rate
```

Comment:
ah, that makes sense. done!

Reply:
Yes, since this function is written specifically for a single model your solution should work (I didn't look at the model def before, hence my more general suggestion). Just to make sure, we do not want to set the dropout rate for `conformer_model.FeedForwardModule.dropout`?

Reply:
right, we just want to modify `feed_forward_residual_dropout_rate`. we should clean up the librispeech models/configs to bake in all the defaults (we won't be modifying them anyways)